docs/codeReview.md

```diff
@@ -0,0 +1,32 @@
+# Code review policy
+
+The goal is to help reviewers to make their intentions clear to the author and for the author to know exactly what is expected of to pass the next review round.
+Review progress is tracked in the [Sprint Board](https://github.com/orgs/ParabolInc/projects/1)
+
+## Reviewer
+
+- Prefix each comment with a label. The labels are not points and will not be summed up or similar.
+    - -2 there is a fundamental design issue
+    e.g. change will crash, high impact performance, high impact on maintainability
+    - -1 please fix this
+    e.g. naming of variable is misleading, coupling of components can be reduced, code is hard to understand
+    - +1 I would do it differently
+      - inconsequential enough to not need another review
+      - some suggestion for the authors consideration, e.g. better variable name, alternative split of components
+      - refactoring of existing code, e.g. While you're in here, could you fix xyz?
+    - +2 kudos
+    e.g. nice work here, I learnt something, good find
+- Final review is approval if there are no negative comments, otherwise change requested
+    - Changes requested? Move ticket to self-review
+    - Approved? Move to maintainer-review
+    - Approved by the maintainer
+      - lowest score +1? the PR goes back to self-review
```

Comment:
+1 I'd avoid using the word score as it implies the numbers are being summed. Perhaps, "if there are no -2 or -1 comments, but there are +1 comments, the PR goes back to self-review"?

On the line below, we could say "if there are no comments or only +2 comments, the PR gets merged"?